src/common/graphQLClient.ts

```diff
@@ -0,0 +1,7 @@
+import { GRAPHQL_API_URL } from './constants'
+import { ApolloClient, InMemoryCache } from '@apollo/client'
```

Comment:
lets test how much this adds to the sdk bundle size; given that we only make a graphql call and dont need complex state management it could make sense to just use fetch

Reply:
@kopy-kat  yeah make sense, updated to use fetch